pilot/pkg/serviceregistry/memory/discovery_mock.go

```diff
@@ -43,26 +43,29 @@ var (
 
 	// HelloProxyV0 is a mock proxy v0 of HelloService
 	HelloProxyV0 = model.Proxy{
-		Type:        model.SidecarProxy,
-		IPAddresses: []string{HelloInstanceV0},
-		ID:          "v0.default",
-		DNSDomain:   "default.svc.cluster.local",
+		Type:            model.SidecarProxy,
+		IPAddresses:     []string{HelloInstanceV0},
+		ID:              "v0.default",
+		DNSDomains:      []string{"default.svc.cluster.local"},
+		ConfigNamespace: "default",
```

Comment:
I found this change to be a little troubling.  context_test.go was failing without it - because it was expecting the in-memory database to match the new calculation of comparison of got/vs/want for role.ConfigNamespace...

Reply:
I also find it troubling. If ADS was populating this field and you removed that line of code, is it still being set?